registry/registry.go

```diff
@@ -339,32 +343,37 @@ func (r *Registry) collectExternalDependenciesFromData(filesData map[string]*dat
 				base := fileData.TSFileName
 				target := data.GetTSFileName(typeInfo.File)
 				sourceFile := ""
-				foundAtRoot, alias, err := r.findRootAliasForPath(func(absRoot string) (bool, error) {
-					completePath := filepath.Join(absRoot, typeInfo.File)
-					_, err := os.Stat(completePath)
-					if err != nil {
-						if os.IsNotExist(err) {
-							return false, nil
+				if pkg, ok := r.TSPackages[target]; ok {
+					log.Debugf("pkg: %s", pkg)
```

Comment:
Could we have a better debug message? saying `package import override %s has been found for file%s `

Reply:
Thank you for your review. I will fix this ASAP.

Reply:
@lyonlai I fixed the debug message.

Reply:
I think google-protobuf package is good for testing. I try to write a test. Please wait for a while.

Reply:
@lyonlai I added integration test for ts_package option.

Reply:
nice.